plugin/src/main/java/com/denizenscript/denizen/events/item/ItemRecipeFormedScriptEvent.java

```diff
@@ -56,47 +67,21 @@ public boolean matches(ScriptPath path) {
         return super.matches(path);
     }
 
-    @Override
-    public boolean applyDetermination(ScriptPath path, ObjectTag determinationObj) {
-        if (determinationObj.canBeType(ItemTag.class)) {
-            ItemTag result = determinationObj.asType(ItemTag.class, getTagContext(path));
-            event.getInventory().setResult(result.getItemStack());
-            return true;
-        }
-        else {
-            return super.applyDetermination(path, determinationObj);
-        }
-    }
-
     @Override
     public ScriptEntryData getScriptEntryData() {
         return new BukkitScriptEntryData(EntityTag.getPlayerFrom(event.getView().getPlayer()), null);
     }
 
     @Override
     public ObjectTag getContext(String name) {
-        switch (name) {
-            case "item": return result;
-            case "inventory": return InventoryTag.mirrorBukkitInventory(event.getInventory());
-            case "recipe": {
-                ListTag recipe = new ListTag();
-                for (ItemStack itemStack : event.getInventory().getMatrix()) {
-                    if (itemStack != null && itemStack.getType() != Material.AIR) {
-                        recipe.addObject(new ItemTag(itemStack));
-                    }
-                    else {
-                        recipe.addObject(new ItemTag(Material.AIR));
-                    }
-                }
-                return recipe;
-            }
-            case "recipe_id":
-                if (event.getRecipe() instanceof Keyed) {
-                    return new ElementTag(((Keyed) event.getRecipe()).getKey().toString());
-                }
-                break;
-        }
-        return super.getContext(name);
+        return switch (name) {
+            case "item" -> result;
+            case "inventory" -> InventoryTag.mirrorBukkitInventory(event.getInventory());
+            case "recipe" -> new ListTag(Arrays.asList(event.getInventory().getMatrix()), ItemTag::new);
+            case "recipe_id" -> event.getRecipe() instanceof Keyed ? new ElementTag(((Keyed) event.getRecipe()).getKey().toString()) : null;
+            case "is_repair" -> new ElementTag(event.isRepair());
+            default -> null;

```

Comment:
You misunderstood what I said earlier - you still need the `super` call, just not within the handling for another context.